operator/src/test/java/oracle/kubernetes/operator/steps/ServerDownIteratorStepTest.java

```diff
@@ -0,0 +1,383 @@
+    return objectMeta
+            .name(LegalNames.toPodName(UID, serverName))
+            .putLabelsItem(LabelConstants.SERVERNAME_LABEL, serverName);
+  }
+
+  /**
+   * Setup env for tests.
+   * @throws NoSuchFieldException if TestStepFactory fails to install
+   */
+  @Before
+  public void setUp() throws NoSuchFieldException {
+    mementos.add(TestUtils.silenceOperatorLogger().ignoringLoggedExceptions(ApiException.class));
+    mementos.add(TuningParametersStub.install());
+    mementos.add(testSupport.install());
+
+    testSupport.defineResources(domain);
+    testSupport
+            .addToPacket(ProcessingConstants.DOMAIN_TOPOLOGY, domainConfig)
+            .addDomainPresenceInfo(domainPresenceInfo);
+    testSupport.addComponent(
+            ProcessingConstants.PODWATCHER_COMPONENT_NAME,
+            PodAwaiterStepFactory.class,
+            new PodHelperTestBase.DelayedPodAwaiterStepFactory(1));
+
+  }
+
+  /**
+   * Cleanup env after tests.
+   * @throws Exception if test support failed
+   */
+  @After
+  public void tearDown() throws Exception {
+    for (Memento memento : mementos) {
+      memento.revert();
+    }
+
+    testSupport.throwOnCompletionFailure();
+  }
+
+  @Test
+  public void withConcurrencyOf1_bothClusteredServersShutdownSequentially() {
+    configureCluster(CLUSTER).withMaxConcurrentShutdown(1).withReplicas(1);
+    addWlsCluster(CLUSTER, 8001, MS1, MS2);
+    domainPresenceInfo = createDomainPresenceInfoWithServers(MS1, MS2);
+    testSupport
+            .addDomainPresenceInfo(domainPresenceInfo);
+
+    invokeStepWithServerShutdownInfos(createServerShutdownInfosForCluster(CLUSTER,MS1, MS2));
+    MatcherAssert.assertThat(
+            domainPresenceInfo.isServerPodBeingDeleted(MS2), is(Boolean.TRUE));
+    MatcherAssert.assertThat(
+            domainPresenceInfo.isServerPodBeingDeleted(MS1), is(Boolean.FALSE));
+    testSupport.setTime(10, TimeUnit.SECONDS);
+    MatcherAssert.assertThat(
+            domainPresenceInfo.isServerPodBeingDeleted(MS1), is(Boolean.TRUE));
+  }
+
+  @Test
+  public void withConcurrencyOf2_bothClusteredServersShutdownConcurrently() {
+    configureCluster(CLUSTER).withMaxConcurrentShutdown(2).withReplicas(1);
+    addWlsCluster(CLUSTER, PORT, MS1, MS2);
+    domainPresenceInfo = createDomainPresenceInfoWithServers(MS1, MS2);
+    testSupport
+            .addDomainPresenceInfo(domainPresenceInfo);
+
+    invokeStepWithServerShutdownInfos(createServerShutdownInfosForCluster(CLUSTER,MS1, MS2));
+    MatcherAssert.assertThat(
+            domainPresenceInfo.isServerPodBeingDeleted(MS2), is(Boolean.TRUE));
+    MatcherAssert.assertThat(
+            domainPresenceInfo.isServerPodBeingDeleted(MS1), is(Boolean.TRUE));
+  }
+
+  @Test
+  public void withConcurrencyOf0_clusteredServersShutdownConcurrently() {
+    configureCluster(CLUSTER).withMaxConcurrentShutdown(0);
+    addWlsCluster(CLUSTER, PORT, MS1, MS2);
+
```

Comment:
Remove blank lines in the middle of your test setup, reserving them as dividers between setup, execution, and validation.